src/mobot_client/mobot.py

```diff
@@ -1,5 +1,6 @@
 # Copyright (c) 2021 MobileCoin. All rights reserved.
 
+## CLASS IS DEPRECATED --- HERE AS A REFERENCE ONLY
```

Comment:
Hmm... should we just delete it then? We can always refer to it in git history if need be.